pkg/ee/conversion/helm.go

```diff
@@ -307,6 +308,10 @@ func ConvertDatacenters(datacenterMeta map[string]provider.DatacenterMeta, globa
 		seed.Kind = "Seed"
 		seed.Namespace = targetNamespace
 
+		if pauseProvisioning {
+			seed.Annotations[common.SkipReconcilingAnnotation] = "true"
```

Comment:
Consistently with my previous comment I would add an empty string, as I think that the discriminant should be the existence of the annotation more than its value

Reply:
@irozzo-1A done